### ISSUE TITLE
Add support for node_module paths in the `@design-tokens` rule

### ISSUE THREAD

**Issue by fredboyle** [csstools/postcss-plugins#587]
Is there a recommended PostCSS plugin or method to allow the use of node_module paths in the `@design-tokens` rule?

Example:
```css
@design-tokens url('@org/module-name/dist/json/tokens.json') format('style-dictionary3');
```

**Comment by romainmenke** [comment]
The only way to currently do this is to write out the full path to your `node_modules` folder.

An example of this can be found in this test : 

https://github.com/csstools/postcss-plugins/blob/main/plugins/postcss-design-tokens/test/imported.css

**Comment by fredboyle** [comment]
Ugh that's a bummer. Thanks.

**Comment by romainmenke** [comment]
@fredboyle We might be able to get this to work with this format :

```css
@design-tokens url('node_modules://@org/module-name/dist/json/tokens.json') format('style-dictionary3');
```

The `node_modules://` scheme is needed because we need a way to detect that this "import" to traverse `node_modules` folders in search of a package.

I considered `npm://` but that is misleading because the files are not loaded from npm directly.

Would this solve your issue?

**Comment by fredboyle** [comment]
@romainmenke Seems reasonable to me and not much of a burden on developers. Makes things a bit more universal and without the need to calculate relative paths every time.

**Comment by romainmenke** [comment]
@fredboyle We have released `v1.2.0` which should resolve this : https://github.com/csstools/postcss-plugins/blob/main/plugins/postcss-design-tokens/CHANGELOG.md#120-september-7-2022

Can you test this version to make sure it works as expected?

_If you happen to use the [VSCode extension](https://marketplace.visualstudio.com/items?itemName=RomainMenke.csstools-design-tokens), that one has also been updated with support for node_modules paths._

**Comment by romainmenke** [comment]
Closing because I assume this now works correct.
If you do encounter any more issues feel free to open a new issue or to post a comment here.

Thank you again using the packages and for giving feedback!
